arm/compute/containerservice/client.go

```diff
@@ -1,44 +0,0 @@
-// Package compute implements the Azure ARM Compute service API version
-// 2016-03-30.
```

Comment:
@garimakhulbe Just FYI: please let @mcardosos & @jhendrixMSFT know who is the point of contact from core team (who knows about this) tracking this so that they know whom to talk to.